docs/csharp/language-reference/compiler-messages/cs0052.md

```diff
@@ -49,4 +49,4 @@ public class MainClass
 - [C# Keywords](../keywords/index.md)
 - [Access Modifiers](../keywords/access-modifiers.md)
 - [Accessibility Levels](../keywords/accessibility-levels.md)
-- [Modifiers](../keywords/modifiers.md)
+- [Modifiers](/dotnet/csharp/language-reference/keywords)
```

Comment:
I'm really on the fence about these links because they used to go to a farm link page for this node:

![image](https://user-images.githubusercontent.com/12971179/67928077-6541c580-fb77-11e9-904e-d1914aa3784a.png)

When these pages were removed because of the TOC accessibility changes, we can no longer send folks to that node. So either this has to be changed to say Keywords or the link should take you to the first link on that TOC which would be the same link as Access Modifiers.

Reply:
As part of #10744, all the modifiers will be on one page. I think the current link is OK, keeping the link text as "modifiers" will help us find this and redirect it to the new modifiers page.